### DISCUSSION TITLE
Implement a Temporary Ignore (/tempignore) command.

### DISCUSSION THREAD

**Discussion by accLarson** [EssentialsX/Essentials#5185] [Ideas and Feature Suggestions]
A player on my server suggested an idea I thought was worth passing on to EssentialsX.

Something like `/tempignore <player> <hourcount>`

That player's words "You can ignore someone and just unignore later, but sometimes you only want to ignore someone during the conversation they are currently having, not in general. It’s tough to know when that conversation ends, sort of like muting a discord channel for 15 minutes or whatever."

It's fair to say that many players remain unintentionally ignored when the intent was to unignore but was forgotten. 

Might also be nice to create a `/ignorelist` at the same time for clarity. (`/ignore` does display a list of players you currently ignore but I find this isn't very intuitive for players`)

Thanks for reading!

**Comment by Evidentsinger14** [comment]
I like the idea. Although, instead of creating a new command for it, there could be another argument in the already existing ignore command. ex: `/ignore JRoy` is permanent, but you can also do `/ignore JRoy 15m`

**Reply by accLarson** [reply]
That would be great
